.github/ci/files/docker-compose.yaml

```diff
@@ -0,0 +1,55 @@
+services:
+    redis:
+        image: redis:alpine
+        command: [ redis-server, --maxmemory, 128mb, --maxmemory-policy, volatile-lru, --save, '""' ]
+
+    db:
+        image: mysql:latest
+        working_dir: /application
+        command: [ mysqld, --log_bin_trust_function_creators=1, --character-set-server=utf8mb4, --collation-server=utf8mb4_unicode_ci, --innodb-file-per-table=1 ]
```

Comment:
only these are the changes from base docker compose, beside the volume name change.
https://github.com/pimcore/demo/actions/runs/6150374522/job/16688185233 experienced a problem that only mysql configuration is triggering, which is later on fixed by adding `log_bin_trust_function_creators` here